pkg/component/controller/kuberouter.go

```diff
@@ -272,6 +273,7 @@ spec:
           - mountPath: /etc/kube-router
             name: kube-router-cfg
       hostNetwork: true
+      hostPID: true
```

Comment:
This is necessary due to [some upstream fixes](https://github.com/cloudnativelabs/kube-router/pull/1582) to hairpinning. I do know that we are testing hairpinning in the inttests, bit from the fix's description it sounds as it has been broken before?

> With this fix, kube-router now passes the Kubernetes end-to-end conformance test for hairpinning traffic.

:thinking: